### PR TITLE
Feature/front end all inventory list

### PR DESCRIPTION
**Pull request by benmac11**
I'm still working on this ticket. I'm having some issues. I realized my develop was a bit behind on updates. I tried to pull develop so it would be up to date. It told me I had to commit my changes first.

### CODE REVIEW

File: client/src/components/inventory/inventoryHeading.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import './inventory.scss';
+
+class inventoryHeading extends Component {
+    render() {
+        return (
+            <div>
+                <h2>Inventory</h2>
```
Comment (drunyon88) [inline client/src/components/inventory/inventoryHeading.js]: should be h1, then h1 mobile in css file

File: client/src/components/inventory/inventory.js [hunk 2]
```diff
@@ -13,14 +13,22 @@ class Inventory extends Component {
     })};
 
     render() {
-        return this.state.map(item =>(
+        return this.state.inventory.map(item =>(
             <div>
-                <h2>Inventory</h2>
-                <input type="text" placeholder="Search"/>
-                <p>{item}</p>
+                <p>ITEM</p>
```
Comment (drunyon88) [inline client/src/components/inventory/inventory.js]: shouldnt be all p tags, need classnames

File: client/src/components/inventory/inventoryHeading.js (hunk 1 above)
Comment (benmac11) [reply client/src/components/inventory/inventoryHeading.js]: Yeah I'm not even close to being done. Im just trying to pull develop but it wont let me unless I pushed my changes

File: client/src/components/inventory/inventory.js (hunk 2 above)
Comment (benmac11) [reply client/src/components/inventory/inventory.js]: Yeah I know I'm just getting the data on the page

**Comment by arshad1416** [comment]
Ben, I've merged it for you so that you can get the updated branch.

**Pull request merged**